velox/dwio/parquet/reader/duckdb/Allocator.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/common/memory/Memory.h"
+#include "velox/external/duckdb/duckdb.hpp"
+
+namespace facebook::velox::duckdb {
+
+struct PrivateVeloxAllocatorData : public ::duckdb::PrivateAllocatorData {
+  PrivateVeloxAllocatorData(memory::MemoryPool& pool_) : pool(pool_) {}
+
+  ~PrivateVeloxAllocatorData() override {}
+
+  memory::MemoryPool& pool;
+};
+
+::duckdb::data_ptr_t velox_pool_allocate(
```

Comment:
naming: It is confusing to mix-and-match different naming scheme. I wonder if we could stick to Velox naming conventions unless we have to match DuckDB's, e.g. when overriding methods, etc.

Reply:
Yeah, we can use Velox naming here. Fixed.